data/data/baremetal/variables-baremetal.tf

```diff
@@ -1,91 +1,49 @@
-variable "ironic_uri" {
+variable "IronicURI" {
```

Comment:
So that's part of the hcl renderer, not sure on how to change it. I had to stop using JSON renderer because of the referenced bug. Instead, i used that project that renders to HCL format automatically, but it generates the vars with upper case... what should i do there?

Reply:
I think you should be able to use the annotation support in go, `hcl:"ironic_uri"`

Reply:
See the example here: https://github.com/rodaine/hclencoder